### PR TITLE
Update docstings and arg names of InterpolatedVolumeParameter and InterpolatedFlowParameter

### PR DESCRIPTION
**Pull request by Batch21**

### CODE REVIEW

**Comment by jetuk** [comment]
@Batch21 what have you done here?

**Comment by Batch21** [comment]
oops, I created the branch from the wrong location. Should be fixed after a bit of Git fun.

**Comment by jetuk** [comment]
Shall we close this then?

**Comment by Batch21** [comment]
Is it not OK as its now branched correctly from master?

**Comment by jetuk** [comment]
Oh I missed the force push, sorry.

File: pywr/parameters/parameters.py [hunk 1]
```diff
@@ -134,11 +146,22 @@ def load(cls, model, data):
...
 class InterpolatedFlowParameter(AbstractInterpolatedParameter):
     """
-    Generic interpolation parameter that uses a node's  flow at the previous time-step for interpolation.
+    Generic interpolation parameter that uses a node's flow at the previous time-step for interpolation.
 
+    Parameters
+    ----------
+    node: Node
+        Node to provide input flow values to interpolation caluculation
+    volumes: array_like
```
Comment (jetuk) [inline pywr/parameters/parameters.py:155]: flows.

**Review by jetuk — CHANGES_REQUESTED** [review]
Can we rename the variable in `InterpolatedFlowParameter.load` it currently sets the flows key in a volumes variable which is just confusing.

**Review by jetuk — APPROVED** [review]

**Pull request merged**